### PR TITLE
Add database metrics credentials actions

### LINKED ISSUE #323: Add missing support for Database Cluster Metrics Credentials
Add support for two missing database cluster metrics credentials operations defined here:

  * Get https://docs.digitalocean.com/reference/api/api-reference/#operation/databases_get_cluster_metrics_credentials
  * Update https://docs.digitalocean.com/reference/api/api-reference/#operation/databases_update_cluster_metrics_credentials

There is some existing support for database clusters, so you'll need to extend the implementation for the relevant files in `lib/droplet_kit/mappings`, `lib/droplet_kit/models`, and `lib/droplet_kit/resources`. After that, you'll need to update the documentation for these new methods in the `README.md` file.

To get started, defining operations typically starts with the `resources` file, which for database operations is `lib/resources/database_resource.rb`. You'll add two new actions:

```ruby
action :get_metrics_credentials, 'GET /v2/databases/metrics/credentials' do
  # TODO
end

action :update_metrics_credentials, 'PUT /v2/databases/metrics/credentials' do
  # TODO
end
```

And implement the body and status code handling from there.

### PR DESCRIPTION
**Pull request by caiohsramos**
Closes #323 

References
- Get https://docs.digitalocean.com/reference/api/api-reference/#operation/databases_get_cluster_metrics_credentials
- Update https://docs.digitalocean.com/reference/api/api-reference/#operation/databases_update_cluster_metrics_credentials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ Actions supported:
 * `client.databases.get_eviction_policy(id: 'id')`
 * `client.databases.set_sql_mode(database_sql_mode, id: 'id')`
 * `client.databases.get_sql_mode(id: 'id')`
+* `client.databases.get_metrics_credentials`
+* `client.databases.update_metrics_credentials`
```
Comment (bentranter) [inline README.md:204]: One small change – the `update_metrics_credentials` takes an argument, so we need to include that in the README docs:
```suggestion
* `client.databases.update_metrics_credentials(database_metrics_credentials)`
```

**Review by bentranter — CHANGES_REQUESTED** [review]
One tiny piece of feedback about the README doc change, but the implementation is perfect. Once that's addresses, I'll approve and merge. Thanks again @caiohsramos!

File: README.md (hunk 1 above)
Comment (caiohsramos) [reply README.md:204]: My bad, missed this one!

**Comment by caiohsramos** [comment]
@bentranter updated readme and rubocop file. Should be good now!

**Pull request merged**